kimchi/src/circuits/polynomials/keccak/circuitgates.rs

```diff
@@ -0,0 +1,211 @@
+//~ The keccak gadget is comprised of _ circuit gates (KeccakXOR, .., and Zero)
```

Comment:
Please link this up in `book/specifications/kimchi/Specification.toml` and `book/specifications/kimchi/template.md`